product-service/src/functions/migrateSchema/handler.ts

```diff
@@ -0,0 +1,46 @@
+import "source-map-support/register";
+
+import { Client } from "pg";
+import { winstonLogger } from "@libs/winstonLogger";
+import {
+  formatSuccessResponse,
+  formatErrorResponse,
+} from "@libs/apiResponseBuilder";
+
+export const migrateSchema = (client: Client) => async (event, _context) => {
+  try {
+    await client.query('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";');
+    await client.query(`
+            CREATE TABLE IF NOT EXISTS products (
+                id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+                count INT,
+                date TEXT,
+                description TEXT,
+                image TEXT,
+                location TEXT,
+                price INT,
+                title TEXT
```

Comment:
Title should be **not null**
![image](https://user-images.githubusercontent.com/42437614/115870859-81f74a80-a448-11eb-81f6-e8a62b17c393.png)